script.service.ltvpl/keymapper.py

```diff
@@ -0,0 +1,175 @@
+#keymapper.py
+#
+# Copyright (C) 2018 John Moore
+# Portions Copyright (C) 2014 Thomas Amland
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import xml.etree.ElementTree as ET
+import shutil
+import os
+from threading import Timer
+from time import sleep
+from xbmcgui import Dialog, WindowXMLDialog
+import xbmcaddon, xbmc
+
+from util import GETTEXT, KEYMAPS_USERDATA_FOLDER, ADDONID, ADDON, ACTIVATIONKEY, PYVER
+from resources.lib.Utilities.DebugPrint import DbgPrint
+
+__Version__ = "1.0.1"
+
+keymapfile = os.path.join(KEYMAPS_USERDATA_FOLDER,"gen.xml")
```

Comment:
sorry this is not allowed in the repository. You can't edit such xml files directly from python. The only alternative is to provide instructions on your addon on how to use keymap editor addon.

Reply:
My add-on requires the [**script.module.enum34]** (PR 1108) module in order to run. The enum module is not included in Python 2 but is included in Python 3. This PR was closed recently because no add-on in the repository references it. 

It is my intention that this version of my add-on will only be qualified to run under Leia. I will be removing the requirement for the [**script.module.enum34]** module for Matrix since it will be running Python 3.

Reply:
Removed